### ISSUE TITLE
Upgrading versions causes DLL error in .NET Framework project

### ISSUE THREAD

**Issue by WarrenLevy** [ScottPlot/ScottPlot#4620]
I have just upgraded ScottPlot from v5.0.43 to v5.0.47 and now my code which used to work gives an error. I have a C# Windows application using .NET Framework 4.8.1

I instantiate with the code below
```cs
FormsPlot fp = new FormsPlot() { Dock = DockStyle.Fill };
```
but that line now gives the error below

![image](https://github.com/user-attachments/assets/c70149e1-33c3-4514-afdf-2ed9d3a9aec4)

**Comment by swharden** [comment]
Hi @WarrenLevy, I suspect this is because the DLL for the expected version of SkiaSharp isn't being located at runtime.

I'd clear all the binary folders and try a clean rebuild, probably with a restart of Visual Studio too.

If this is a ScottPlot error that can be reproduced by creating a new .NET Framework 4.8.1 project from scratch let us know and we can open this issue back up and take a closer look 👍

**Comment by WarrenLevy** [comment]
Hi Scott

I created a fresh Windows application and installed v5.0.51 (as opposed to v5.0.47 of before) of ScottPlot.Winforms, then added the one line of code as per the screenshot. I get exactly the same error as in my main application, so I believe that this needs to be looked at.

Thanks

Warren


![image](https://github.com/user-attachments/assets/77264d59-905f-42d4-9b85-64c29ede7cc5)

**Comment by WarrenLevy** [comment]
Hi @swharden 

I was looking at issue #3343 which seemed similar, so tried that solution. I copied the x86 version of libSkiaSharp.dll into my runtime folder and it now works. I'm not sure if this helps you better understand what is going wrong.

**Comment by CoderPM2011** [comment]
Have you changed the output path of the executable file?

**Comment by WarrenLevy** [comment]
No, everything has remained the same, apart from trying to update the
version of ScottPlot.

Having copied libSkiaSharp.dll into the executable folder, it all works as
expected. I've done this successfully on 2 laptops

**Comment by CoderPM2011** [comment]
I tried the code you described and it did not throw any anomalies. 

### Environment
- Windows 10 x64
- Visual Studio 2022
- .NET Framework 4.8.1
- ScottPlot.WinForms v5.0.51

Did you upgrade the original project to a .NET SDK-style project?
Maybe the change will have an impact. 🤔
https://learn.microsoft.com/en-us/dotnet/core/project-sdk/overview
